src/object/srv_obj_migrate.c

```diff
@@ -1897,6 +1897,13 @@ migrate_one_epoch_object(daos_handle_t oh, daos_epoch_range_t *epr,
 			/* DER_DATA_LOSS means it can not find any replicas
 			 * to rebuild the data, see obj_list_common.
 			 */
+			if (rc == -DER_DATA_LOSS) {
+				D_DEBUG(DB_REBUILD, "No replicas for "DF_UOID
+					"\n", DP_UOID(arg->oid));
```

Comment:
Hmm, not sure, since this is not real "error". If we use D_ERROR, it might scare users, and also generate a lot of them, if there is a lot single replica object.